types/index.d.ts

```diff
@@ -786,6 +786,8 @@ export interface InfiniteScrollProps {
   element?: React.ElementType;
   loader?: React.ReactNode;
   threshold?: number;
+  listenToScroll?(standardOffset: number, reverseOffset: number): any;
```

Comment:
```suggestion
  listenToScroll?(standardOffset: number, reverseOffset: number): void;
```
I see different definitions for the return of this callback in different places. I think we could settle with `void`. What you think?